### PR TITLE
Add mem-stat support to Node-Jerry

### PR DESCRIPTION
**Pull request by kisbg**
JerryScript-DCO-1.0-Signed-off-by: bence gabor kis kisbg@inf.u-szeged.hu

### CODE REVIEW

File: deps/jerry/jerryscript.gyp [hunk 1]
```diff
@@ -52,6 +52,8 @@
         'JERRY_GLOBAL_HEAP_SIZE=16384',
         'JERRY_LINE_INFO=1',
         'JERRY_DEBUGGER=0',
+        'JERRY_MEM_STATS=0',
+        'JERRY_LOGGING=1',
```
Comment (zherczeg) [inline deps/jerry/jerryscript.gyp]: Can it copy the value of JERRY_MEM_STATS?

Comment (kisbg) [reply deps/jerry/jerryscript.gyp]: Good Idea!

File: deps/jerry/v8jerry/v8jerry_isolate.cpp [hunk 2]
```diff
@@ -30,7 +30,16 @@ static void error_object_created_callback (const jerry_value_t error_object, voi
...
+    jerry_init_flag_t flag = (jerry_init_flag_t) JERRY_INIT_EMPTY;
+#if (defined JERRY_MEM_STATS && JERRY_MEM_STATS)
+    jerry_port_default_set_log_level (JERRY_LOG_LEVEL_DEBUG);
+    flag = (jerry_init_flag_t) (JERRY_INIT_MEM_STATS | JERRY_INIT_EMPTY);
+#endif
+    jerry_init(flag);
+#if DEBUG_PRINT
+    JerryxHandlerRegisterGlobal((const jerry_char_t *)"print", jerryx_handler_print);
+#endif
+    JerryxHandlerRegisterString((const jerry_char_t *)"normalize", jerryx_handler_string_normalize);
```
Comment (zherczeg) [inline deps/jerry/v8jerry/v8jerry_isolate.cpp]: I think these are registered elsewhere.

**Review by zherczeg — CHANGES_REQUESTED** [review]

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**